player/tests.py

```diff
@@ -1,28 +1,31 @@
-from django.contrib.auth import get_user_model
 from .models import Player, BallGame
 from django.core.exceptions import ValidationError
+from django.contrib.auth.models import User
+from django.db import IntegrityError
 import pytest
 
 
 @pytest.mark.django_db
 class TestPlayerModel:
 
     @pytest.fixture
-    def user(self):
-        return get_user_model().objects.create_user(
-            username='testuser',
-            password='testpass'
-        )
-
-    @pytest.fixture
-    def saved_player(self, user):
-        player = Player.objects.create(
-            user=user,
+    def saved_player(self):
+        player = Player.create(
+            username="daniel",
+            password="password",
             birth_date='1990-01-01',
-            favorite_ball_game=BallGame.Soccer
-        )
+            favorite_ball_game=BallGame.Soccer)
         return player
 
+    def test_create_player_with_new_user(self):
```

Comment:
What will happen if the user already exist? It's up to you to decide what's the expected behavior, but you need a test case that shows it's working as expected.

Reply:
If there is a user with a username equal to the username sent to the Player.create,  IntegrityError will raised.